conf/app.routes

```diff
@@ -36,16 +36,16 @@ GET         /cases/:reference/reassign-case                     @uk.gov.hmrc.tar
 POST        /cases/:reference/reassign-case                     @uk.gov.hmrc.tariffclassificationfrontend.controllers.ReassignCaseController.reassignCase(reference: String, origin: String)
 GET         /cases/:reference/reassign-case/confirmation        @uk.gov.hmrc.tariffclassificationfrontend.controllers.ReassignCaseController.confirmReassignCase(reference: String, origin: String)
 
-GET         /cases/:reference/refer                             @uk.gov.hmrc.tariffclassificationfrontend.controllers.ReferCaseController.getReferCase(reference: String, activeTab: Option[String] ?= None)
-POST        /cases/:reference/refer                             @uk.gov.hmrc.tariffclassificationfrontend.controllers.ReferCaseController.postReferCase(reference: String, activeTab: Option[String] ?= None)
+GET         /cases/:reference/refer                             @uk.gov.hmrc.tariffclassificationfrontend.controllers.ReferCaseController.getReferCase(reference: String, activeTab: Option[uk.gov.hmrc.tariffclassificationfrontend.controllers.ActiveTab] ?= None)
```

Comment:
It might read better if we import these packages in build sbt like the exmple: https://github.com/hmrc/secure-data-exchange-management-frontend/blob/d3be8ee2af88bd5fcd4b906befdb5bdac08c881c/build.sbt#L28